src/main/java/com/revature/chronicle/controller/TicketController.java

```diff
@@ -0,0 +1,174 @@
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.chronicle.models.Notification;
+import com.revature.chronicle.models.Ticket;
+import com.revature.chronicle.models.User;
+import com.revature.chronicle.services.NotificationService;
+import com.revature.chronicle.services.TicketService;
+
+@RestController(value="ticketController")
+@RequestMapping(path="/ticket")
+public class TicketController {
+	
+	@Autowired
+	private TicketService ticketService;
+	
+	@Autowired
+	private NotificationService notificationService;
+	
+	//this endpoint returns all notifications for the trainer
+	@GetMapping(path="notifications")
+	public ResponseEntity <List<Ticket>> getNotifications(HttpServletRequest req){
+		 User user =  (User) req.getAttribute("user");
+		 List<Ticket> tickets = ticketService.ticketsByEditor(user);
+		 return new ResponseEntity<>(tickets, HttpStatus.OK);
+	}
+
+	
+	//this endpoint returns a list of submitted tickets
+	@GetMapping(path="submitted")
+	public ResponseEntity <List<Ticket>> findAllSubmittedTickets(){
+		List<Ticket> tickets =  this.ticketService.findAll();
+		return new ResponseEntity<>(tickets, HttpStatus.OK);
+	}
+	
+	//this endpoint returns a list of all pending tickets
+	@GetMapping(path="pending")
+	public ResponseEntity <List<Ticket>> findAllPending(){
+		 List<Ticket> tickets = ticketService.ticketsByStatus("PENDING");
+		 return new ResponseEntity<>(tickets, HttpStatus.OK);
+	}
+	
+	//this endpoint returns a list of all under review tickets
+	@GetMapping(path="underreview")
+	public ResponseEntity <List<Ticket>> findAllUderReview(){
+		
+			 List<Ticket> tickets = ticketService.ticketsByStatus("UNDER_REVIEW");
+			 return new ResponseEntity<>(tickets, HttpStatus.OK);
+		}
+			
+	
+	//this endpoint returns a list of all under review tickets
+		@GetMapping(path="ticketsforeditor")
+		public ResponseEntity <List<Ticket>> findAllTicketsByEditor(HttpServletRequest req){
+			
+				 User user =  (User) req.getAttribute("user");
+				 List<Ticket> tickets = ticketService.ticketsByEditor(user);
+				 return new ResponseEntity<>(tickets, HttpStatus.OK);
+			}
+	
+	
+	
+	
+	//this endpoint saves array of tickets
+		@PostMapping(path="saveall", consumes = MediaType.APPLICATION_JSON_VALUE)
+		public boolean saveAll(@RequestBody List<Ticket> tickets) {
+			Date dateIssued =new Date(System.currentTimeMillis());
+			System.out.println("save all method");
+			for(Ticket t: tickets) {
+				t.setDateIssued(dateIssued);
+			}
+			return this.ticketService.saveAll(tickets);
+		}
+	
+		
+		
+		
+	//this endpoint updates a ticket by the editor
+		@PostMapping(path="update", consumes = MediaType.APPLICATION_JSON_VALUE)

```

Comment:
Generally speaking, you do not want to use verbs in the resource name. Please change these resource names.